make/config/istio-csr-values.yaml

```diff
@@ -1,4 +1,4 @@
-replicaCount: 3
```

Comment:
note: this is technically an unrelated change. I reduced the count by 1 here because istio-csr is leader-elected and the spare pods aren't really used in the e2e tests we do at all.

I think there's some value in having a spare running, but I couldn't see a reason to have _two_ spares running!